File: javascript/binarySearch.test.js
const binarySearch = require('./binarySearch')

test('Test #1', ()=>{
  const test1 = [0,1,21,33,44,45,66,77,88,99]
  expect(binarySearch(test1, 33)).toEqual(3)
})

test('Test #2', ()=>{
  const test1 = [0,1,21,33,44,45,66,77,88,99]
  expect(binarySearch(test1, 77)).toEqual(7)
})

File: python/binarySearch_test_.py
import unittest
from binarySearch import binarySearch

class TestProgram(unittest.TestCase):
  def test_case1(self):
    test1 = [0,1,2,3,4,5,6,7,8]
    self.assertEqual(binarySearch(test1, 2) ,2)

  def test_case2(self):
    test1 = [0,1,2,3,4,5,6,7,8]
    self.assertEqual(binarySearch(test1, 6) ,6)

if __name__ == "__main__":
  unittest.main()